01-Login/src/app/auth/auth.service.ts

```diff
@@ -35,26 +49,41 @@ export class AuthService {
   }
 
   private setSession(authResult): void {
+    // Set isLoggedIn flag in localStorage
+    localStorage.setItem('isLoggedIn', 'true');
     // Set the time that the access token will expire at
     const expiresAt = JSON.stringify((authResult.expiresIn * 1000) + new Date().getTime());
```

Comment:
Technically, we don't need to stringify this anymore because we're not storing it in local storage. But it should work as expected anyway. It's up to you if you want to change it. 👍